rosidl_generator_cpp/test/test_msg_datatype.cpp

```diff
@@ -18,8 +18,8 @@
 
 
 TEST(Test_rosidl_generator_traits, check_data_type) {
-  ASSERT_EQ(rosidl_generator_traits::data_type<rosidl_generator_cpp::msg::String>(),
+  ASSERT_STREQ(rosidl_generator_traits::data_type<rosidl_generator_cpp::msg::String>(),
```

Comment:
Can you please also swap the arguments (string litteral first) as the "expected" result should be the first one and "actual" the second one

Reply:
Swapped: 90749e68ca7b3759c081dcbe7660358209273b3e